### ISSUE TITLE
Add Python 3.9 support

### ISSUE THREAD

**Issue by brunobord** [peopledoc/workalendar#557]
The next Python release is scheduled for 2020-10-05.
We'll probably have to wait for it to be available on travis, though.

- [x] Make sure `pandas` has a .whl for Python 3.9
- [x] Make sure `numpy` has a .whl for Python 3.9
- [x] ~test travis support~~ travis support is dropped in favor of GH actions.
- [x] Add line to the meta in setup.py
- [x] Changelog
- [x] Contributing doc changes
- [x] README.md changes
- [x] ~Upgrade Python version for flake~ travis related

**Comment by brunobord** [comment]
at the moment, tox is not Python 3.9 ready.

**Comment by ewjoachim** [comment]
Hm ? It seems it is, according to https://github.com/tox-dev/tox/issues/1589

**Comment by brunobord** [comment]
I probably need to dig into this, but here's what my naive attempt resulted in:

```
$ tox --version
3.20.1
$ python --version
Python 3.6.9
$ virtualenv --version
virtualenv 20.0.35
$ tox -re py39
GLOB sdist-make: /home/novapost/Projets/WORKALENDAR/workalendar/setup.py
py39 create: /home/novapost/Projets/WORKALENDAR/workalendar/.tox/py39
ERROR: invocation failed (exit code 1), logfile: /home/novapost/Projets/WORKALENDAR/workalendar/.tox/py39/log/py39-0.log
==================================================================================================== log start ====================================================================================================
RuntimeError: failed to query /usr/bin/python3.9 with code 1 err: 'Traceback (most recent call last):\n  File "/home/novapost/.virtualenvs/TOX/lib/python3.6/site-packages/virtualenv/discovery/py_info.py", line 16, in <module>\n    from distutils import dist\nImportError: cannot import name \'dist\' from \'distutils\' (/usr/lib/python3.9/distutils/__init__.py)\n'

===================================================================================================== log end =====================================================================================================
ERROR: InvocationError for command /home/novapost/.virtualenvs/TOX/bin/python3 -m virtualenv --no-download --python /usr/bin/python3.9 py39 (exited with code 1)
_____________________________________________________________________________________________________ summary _____________________________________________________________________________________________________
ERROR:   py39: InvocationError for command /home/novapost/.virtualenvs/TOX/bin/python3 -m virtualenv --no-download --python /usr/bin/python3.9 py39 (exited with code 1)
```

**Comment by ewjoachim** [comment]
Let's take a moment to have a look together at some point :)

**Comment by brunobord** [comment]
I've also tried to build a virtualenv using Python3.9 and it failed:

```sh
$ mkvirtualenv TOX39 --python=`which python3.9`
Running virtualenv with interpreter /usr/bin/python3.9
Traceback (most recent call last):
  File "/home/novapost/.local/lib/python3.6/site-packages/virtualenv.py", line 25, in <module>
    import distutils.sysconfig
ModuleNotFoundError: No module named 'distutils.sysconfig'
```


I've built a venv using Python3.8, and it failed the as the previous Python3.6-based venv.

Then I tried to install `tox-venv` in my Python3.8 TOX-enabled venv:

```sh
$ tox -re py39
GLOB sdist-make: /home/novapost/Projets/WORKALENDAR/workalendar/setup.py
py39 create: /home/novapost/Projets/WORKALENDAR/workalendar/.tox/py39
ERROR: invocation failed (exit code 1), logfile: /home/novapost/Projets/WORKALENDAR/workalendar/.tox/py39/log/py39-0.log
==================================================================================================== log start ====================================================================================================
Error: Command '['/home/novapost/Projets/WORKALENDAR/workalendar/.tox/py39/bin/python3.9', '-Im', 'ensurepip', '--upgrade', '--default-pip']' returned non-zero exit status 1.

===================================================================================================== log end =====================================================================================================
ERROR: InvocationError for command /usr/bin/python3.9 -m venv py39 (exited with code 1)
_____________________________________________________________________________________________________ summary _____________________________________________________________________________________________________
ERROR:   py39: InvocationError for command /usr/bin/python3.9 -m venv py39 (exited with code 1)
```

**Comment by ewjoachim** [comment]
How did you install python3.9 ?

**Comment by brunobord** [comment]
okay it probably came from my local environment. I can make it work on my machine.

Still, at the moment, we're not ready to prove that workalendar still works on Python 3.9, at least on travis, see the prerequisites added to the issue. Still blocked, wait and see.

**Comment by brunobord** [comment]
Update:

* We'll probably switch to Github Changes, and thus, Travis prerequisites are no longer a blocker,
* Pandas has released wheels for Python 3.9 => https://pypi.org/project/pandas/#files
* Numpy has released wheels for Python 3.9 => https://pypi.org/project/numpy/#files
